core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksStore.java

```diff
@@ -208,18 +208,18 @@ public synchronized void writeToCheckpoint(OutputStream output)
       throw new IOException(e);
     }
 
+    int compressLevel = Configuration.getInt(
```

Comment:
Do you anticipate needing to change the compression level while the master is running? If so, keep it this way, if not, make it a member variable.